qiskit_experiments/database_service/db_analysis_result.py

```diff
@@ -417,6 +422,30 @@ def auto_save(self, save_val: bool) -> None:
             self.save()
         self._auto_save = save_val
 
+    @staticmethod
+    def _format_db_display(value):
```

Comment:
I feel like this should be refactored to be a utility helper function instead of a method in the `DbAnalysisResult` class.

Reply:
It's really only for the DB display of this class though.